src/server/index.test.jsx

```diff
@@ -45,9 +54,10 @@ const mockRouteProps = ({ id, service, isAmp, dataResponse, responseType }) => {
   getRouteProps.mockReturnValue({
     isAmp,
     service,
+    variant,
     route: { getInitialData },
     match: {
-      params: { id, service },
+      params: { id, service, variant: variant ? `/${variant}` : undefined },
```

Comment:
Please add a comment to explain that you've added the leading `/` here to the `variant` since that's how it is captured by our route regex. This is otherwise quite confusing to read. 
Also to add clarity, you can make a mock value and use it here. 
```const mockVariantParam = variant => variant ? `/${variant}` : undefined;```

Reply:
Done :)